docs/FEATURES.md

```diff
@@ -1,32 +1,75 @@
+| Base add-ons                  | Shapes                    | One line of data in [shapes.txt](https://gtfs.org/schedule/reference/#shapestxt)      |   https://gtfs.org/getting_started/features/base_add-ons/#shapes |   
+| Base add-ons               | Transfers                 | One line of data in [transfers.txt](https://gtfs.org/schedule/reference/#transferstxt)     | https://gtfs.org/getting_started/features/base_add-ons/#transfers  |
+| Base add-ons | Frequency-Based Service | One line of data in [frequencies.txt](https://gtfs.org/schedule/reference/#frequenciestxt)    | https://gtfs.org/getting_started/features/base_add-ons/#frequency-based-service  |
+| Fares                    | Fare Products             | One line of data in [fare_products.txt](https://gtfs.org/schedule/reference/#fare_productstxt)  | https://gtfs.org/getting_started/features/fares/#fare-products |                                                                                                                      
+| Fares                    | Fare Media                | One line of data in [fare_media.txt](https://gtfs.org/schedule/reference/#fare_mediatxt) |  https://gtfs.org/getting_started/features/fares/#fare-media |
+| Fares                    | Route-Based Fares         | One **network_id** value in [routes.txt](https://gtfs.org/schedule/reference/#routestxt)<br/>OR<br/>one line of data in [networks..txt](https://gtfs.org/schedule/reference/#networkstxt)   |    https://gtfs.org/getting_started/features/fares/#route-based-fares   |
+| Fares                    | Time-Based Fares          | One line of data in [timeframes.txt](https://gtfs.org/schedule/reference/#timeframestxt)   | https://gtfs.org/getting_started/features/fares/#time-based-fares  |
+| Fares                    | Zone-Based Fares          | One line of data in [areas.txt](https://gtfs.org/schedule/reference/#areastxt)   |     https://gtfs.org/getting_started/features/fares/#zone-based-fares |
+| Fares                    | Fares Transfers           | One line of data in [fare_transfer_rules.txt](https://gtfs.org/schedule/reference/#fare_transfer_rulestxt)    |    https://gtfs.org/getting_started/features/fares/#fares-transfers |
+| Fares                    | Fares V1                  | One line of data in [fare_attributes.txt](https://gtfs.org/schedule/reference/#fare_attributestxt)   |   https://gtfs.org/getting_started/features/fares/#fares-v1 |
+| Pathways                 | Pathways Connections*         | One line of data in [pathways.txt](https://gtfs.org/schedule/reference/#pathwaystxt)     |  https://gtfs.org/getting_started/features/pathways/#pathway-connections |
+| Pathways                 | Pathways Details*         | One value of **max_slope**<br/>OR<br/>**min_width** <br/>OR<br/>**length** <br/>OR<br/>**stair_count** in [pathways.txt](https://gtfs.org/schedule/reference/#pathwaystxt) | https://gtfs.org/getting_started/features/pathways/#pathway-details |
```

Comment:
Same as with Pathway Connections, the name used here should be Pathway Details (with the word "pathways" changing to singular). Also correct in the Feature Migration table below.